documentation/src/main/asciidoc/topics/con_performance_partition_handling.adoc

```diff
@@ -0,0 +1,43 @@
+[id='performance-partition-handling_{context}']
+= Network partitions and degraded clusters
+
+{brandname} clusters can encounter split brain scenarios where subsets of nodes in the cluster become isolated from each other.
+Nodes in a network partition can communicate only with other nodes in that partition.
+When this happens, {brandname} enters **DEGRADED** mode.
+
+{brandname} provides different strategies for handling network partitions.
+The strategy that you choose depends entirely on application demands for data.
+
+Removing a network partition and merging the cluster back together means you need to choose between consistency or availability while {brandname} merges the partitions and resolves conflicts.
+If higher availability is your primary concern, {brandname} provides a merge strategy that allows read and write operations to continue.
+If consistency is more important than availability, you can deny read and write operations.
+There is also an option to allow reads only, which preserves consistency by denying writes to entries and provides availability as applications can still perform reads.
+
+Obviously while the {brandname} cluster is in **DEGRADED** mode, performance is not optimal and there is a cost associated with returning the cluster to a healthy state.
+This is where merge policies come in to resolve any conflicting entries.
+
+By default {brandname} does not attempt to resolve conflicts on merge which means clusters return to a healthy state sooner and there is no performance penalty beyond normal cluster rebalancing.
+However, in this case, data in the cache is more likely to be inconsistent.
+
+To resolve conflicts {brandname} provides the following merge policies:
+
+* `PREFERRED_ALWAYS` means that {brandname} finds the value that exists on the majority of nodes in the cluster and applies it, which could result in restoring older values.
+* `PREFERRED_NON_NULL` means that, if the value is null on the majority of nodes, {brandname} uses the first ... what?
+* `REMOVE_ALL` means that {brandname} always removes conflicting entries, which means you lose all entries that are modified concurrently with different values.
+
+.Garbage collection impacts on partition handling
```

Comment:
I know this section is specifically about Garbage Collection during partition handling, but I think you could add a not in general about garbage collection issues. For example, using more of the heap than is recommended will cause more frequent garbage collection and pauses which could cause cluster instability and network partitions.

Reply:
A note like 'first of all prevent from long GC pauses instead of attempt to fix it with PH'

Reply:
Thanks @alanfx and @wfink This is great detail.